alpha/apps/kaltura/lib/batch2/bcdl/kBusinessPreConvertDL.php

```diff
@@ -2268,4 +2276,71 @@ public static function checkConditionalProfiles($entry, $mediaInfo)
 		KalturaLog::log("None of the conditions are met.");
 		return;
 	}
+
+	/**
+	 * @param $entry
+	 * @return conversionProfile2|null
+	 * @throws Exception
+	 */
+	public static function retrieveConversionProfileByType($entry)
+	{
+		$profile = null;
+		if(PermissionPeer::isValidForPartner(PermissionName::FEATURE_STATIC_CONTENT_CONVERSION, $entry->getPartnerId()))
+		{
+			$key = self::getConversionProfileKey($entry);
+			if ($key && isset(self::$staticContentConditionalMapBySourceType[$key]))
+			{
+				$profile = conversionProfile2Peer::retrieveByPartnerIdAndSystemName($entry->getPartnerId(), self::$staticContentConditionalMapBySourceType[$key], ConversionProfileType::MEDIA, true);
+			}
+		}
+		elseif($entry->getSourceType() == EntrySourceType::LECTURE_CAPTURE)
+		{
+			$profile = conversionProfile2Peer::retrieveByPartnerIdAndSystemName($entry->getPartnerId(), self::$conditionalMapBySourceType[EntrySourceType::LECTURE_CAPTURE], ConversionProfileType::MEDIA);
+		}
+		else
+		{
+			$profile = myPartnerUtils::getConversionProfile2ForEntry($entry->getId());
+		}
+
+		return $profile;
+
+
+	}
+
+	/**
+	 * @param Entry $entry
+	 * @return string|null
+	 */
+	public static function getConversionProfileKey(Entry $entry)
+	{
+		$adminTags = $entry->getAdminTagsArr();
+		if ($entry->getSourceType() == EntrySourceType::LECTURE_CAPTURE)
+		{
+			if (in_array('kalturacapture', $adminTags))
```

Comment:
To preserve the old logic better return self::KALTURA_CAPTURE_CONVERSION_KEY as the default value in this case